src/views/statusView.ts

```diff
@@ -445,23 +450,51 @@ export default class StatusView implements vscode.Disposable {
         }
     }
 
-    public updateStatusBarForEditor(editor: vscode.TextEditor, isConnected: boolean): void {
+    public updateStatusBarForEditor(
+        editor: vscode.TextEditor,
+        connectionInfo: ConnectionInfo,
+    ): void {
         // Change the status bar to match the newly active editor
         if (typeof editor !== "undefined") {
             const fileUri = getUriKey(editor.document.uri);
             const bar = this._statusBars[fileUri];
             if (bar) {
-                if (!isConnected) {
+                if (!connectionInfo) {
+                    /**
```

Comment:
Can you convert these block comments to single-line comments.  I think the generated code is wasting some space by adding those.